scripts/lint-staged/tslint.js

```diff
@@ -1,8 +1,7 @@
 // @ts-check
 
-const execSync = require('../exec-sync');
+const { spawnSync } = require('child_process');
```

Comment:
If our `execSync` utility is broken, shouldn't we remove or fix it rather than just removing one place that uses it?